bin/consequence_mapping/consequence_mapping.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
+
+"""
+Pipeline for mapping variants to the genes they affect and their functional consequences, using Ensembl VEP API.
+For documentation, refer to /docs/consequence-mapping.md
+"""
+
+import itertools
+import json
+import logging
+import os
+import requests
+import sys
+
+from retry import retry
+
+logging.basicConfig()
+logger = logging.getLogger('consequence_mapping')
+logger.setLevel(logging.INFO)
```

Comment:
Suggest moving logging initialization to __init__.py so that it can be shared across other python source files in the directory (should there be any in the future)